### PR TITLE
Identifier contingency list endpoints

### PR DESCRIPTION
**Pull request by etiennehomer**

### CODE REVIEW

File: src/main/java/org/gridsuite/explore/server/ExploreController.java [hunk 1]
```diff
@@ -159,6 +159,30 @@ public ResponseEntity<Void> replaceFilterContingencyListWithScript(@PathVariable
...
+                                                          @RequestParam(QUERY_PARAM_PARENT_DIRECTORY_ID) UUID parentDirectoryUuid,
+                                                          @RequestHeader("userId") String userId) {
+        exploreService.createIdentifierContingencyList(listName, content, description, userId, parentDirectoryUuid);
+        return ResponseEntity.ok().build();
+    }
+
+    @PostMapping(value = "/explore/identifier-contingency-lists")
+    @Operation(summary = "Duplicate a identifier contingency list")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "Identifier contingency list has been created")})
+    public ResponseEntity<Void> createIdentifierContingencyList(@RequestParam("duplicateFrom") UUID parentListId,
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/explore/server/ExploreController.java:177]: duplicate ?

File: src/main/java/org/gridsuite/explore/server/ExploreController.java [hunk 2]
```diff
@@ -159,6 +159,30 @@ public ResponseEntity<Void> replaceFilterContingencyListWithScript(@PathVariable
...
+                                                          @RequestParam("listName") String listName,
+                                                          @RequestParam("description") String description,
+                                                          @RequestParam(QUERY_PARAM_PARENT_DIRECTORY_ID) UUID parentDirectoryUuid,
+                                                          @RequestHeader("userId") String userId) {
+        exploreService.createIdentifierContingencyList(parentListId, listName, description, userId, parentDirectoryUuid);
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/explore/server/ExploreController.java:182]: duplicate ?

Comment (etiennehomer) [reply src/main/java/org/gridsuite/explore/server/ExploreController.java]: Done

File: src/main/java/org/gridsuite/explore/server/ExploreController.java (hunk 1 above)
Comment (etiennehomer) [reply src/main/java/org/gridsuite/explore/server/ExploreController.java]: done

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**